### PR TITLE
Plain javascript

### PR DESCRIPTION
**Pull request by m05e5**
- Create a collection that keeps a list of books
- Create a function to add a new book to the collection, with title and author.
- Create a function to remove a book from the collection
- Display all books saved in the collection in the top part of the page.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+if (localStorage.getItem('data') === null) {
+  localStorage.setItem('data', '[]');
+}
+
+function Book(id, title, author) {
+  this.id = id;
+  this.title = title;
+  this.author = author;
+}
```
Comment (Davidosky007) [inline script.js:11]: * The collection for storing a list of the created books is missing, please fix it as currently, books created are not stored

File: script.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+function addBooks(bookList) {
+  const books = document.getElementById('books');
+  const liToRemove = document.querySelectorAll('li');
+  liToRemove.forEach((item) => {
+    item.remove();
+  });
+
```
Comment (Davidosky007) [inline script.js:18]: * The function addBooks is not adding books to a collection of books eg( An array) 
* i suggest you make this function add a book to a collection and not add books directly to the DOM

File: script.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+function getFromLocalStorage() {
+  return JSON.parse(localStorage.getItem('data'));
+}
+
+function saveToLocalStorage(bookList) {
+  localStorage.setItem('data', JSON.stringify(bookList));
+  addBooks(getFromLocalStorage());
+}
```
Comment (Davidosky007) [inline script.js:50]: * currently books created and removed does not reflect in local storage kindly fix it 🙏

File: script.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+const addBtn = document.getElementById('add');
+addBtn.addEventListener('click', () => {
+  const bookList = getFromLocalStorage();
+  const id = Date.now();
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  const book = new Book(id, title, author);
+  bookList.push(book);
+  saveToLocalStorage(bookList);
+});
```
Comment (Davidosky007) [inline script.js]: * When the add button is clicked this error shows up please fix it 🙏 
Screenshot (144)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @mugberto @m05e5, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
* Kindly check below in the comments section and make the necessary changes 👍 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by mimipeshy — APPROVED** [review]
## Hi Team @m05e5 and @mugberto 👋🏾 👋🏾 , 

Your project is complete! 
#### Highlights

- You have made the changes requested by the previous TSE ✔️ 

 There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**